templates/docker-compose.yml.j2

```diff
@@ -0,0 +1,6 @@
+version: '2'
```

Comment:
I imagine you use version 2 because 3 it's still too young right? http://blog.terranillius.com/post/composev3_swarm/